changed description, added new test cases (#176)
mate-academy/jv-clonable: README.md

# jv-cloneable

> We have a Car class and an Engine class. Define the clone method so that the cloning will deep.

Reviewer: ```suggestion
We have a Car class and an Engine class. Define the clone method so that the cloning will be deep.
```

Add at least 5 fields to each class (Car and Engine). Remember to use the engine in your car.
You also need to override `toString()` method for the car and engine.

Check the `clone ()` method in the `Application` class.

Steps:
1. create an engine;
2. create a car and assign an engine to it;

> 3. call `clone()` method on the car object, assign the result to a new variable;

Reviewer: ```suggestion
3.  Declare carClone variable by calling `clone()` on our existing car;
```

> 4. change one of the parameters (fields) of the motor, which was created at the beginning (1);

Reviewer: ```suggestion
4. change one of the parameters (fields) of the engine in the original car;
```

5. print the parameters of both cars to the console. Compare engines for both cars. 

> If the motor parameters are different, the task is completed correctly.

Reviewer: ```suggestion
If engine parameters differs in original and cloned car, the task is completed correctly, well done!
```
